api/envoy/api/v2/core/address.proto

```diff
@@ -20,6 +20,9 @@ message Pipe {
   // Paths starting with '@' will result in an error in environments other than
   // Linux.
   string path = 1 [(validate.rules).string = {min_bytes: 1}];
+
+  // The mode for the Pipe. Not applicable for abstract sockets.
```

Comment:
Please add a note in version history with a ref link to this new feature.

Reply:
Done